aiosmtpd/controller.py

```diff
@@ -8,13 +8,16 @@
 
 @public
 class Controller:
-    def __init__(self, handler, loop=None, hostname='::0', port=8025):
+    def __init__(self, handler, loop=None, hostname='::0', port=8025,
+                 thread_timeout=5):
```

Comment:
I pushed a patched version which includes the env var. I can remove it you want, but I think it's not entirely useless. For example in CI tests as @warsaw brought up as an example.